superset/charts/schemas.py

```diff
@@ -664,6 +663,10 @@ class ChartDataQueryObjectSchema(Schema):
     timeseries_limit = fields.Integer(
         description="Maximum row count for timeseries queries. Default: `0`",
     )
+    timeseries_limit_metric = fields.Integer(
+        description="Maximum row count for timeseries queries. Default: `0`",
```

Comment:
Good catch 👍 The description should probably be updated to read `Metric used to limit timeseries queries by`.

Reply:
changed